custom/icds_reports/templates/icds_reports/icds_app/mobile/map-or-sector-view.directive.html

```diff
@@ -1,4 +1,5 @@
 <div class="map-or-sector-view-directive">
+    <div id="chartPopup" class="chartPopup nvtooltip"></div>
```

Comment:
the `nvtooltip` class is the one that adds the crazy `pointer-events: none` property, but i wanted to leverage most of the other styling from it...